README.md

```diff
@@ -0,0 +1,34 @@
+# FeedHenry RainCatcher Simple Store
+
+A CRUD storage engine that supports both in-memory and $fh.db backends.
```

Comment:
I hate this last letter, but it seems to be present on the $fh.db docs so let's roll with it :+1: